spec/helpers/solr_facet_response.yaml

```diff
@@ -50,15 +50,17 @@ facet_counts:
     - Non Global
     - 329
     facet_sensor:
-    - Special Sensor Microwave Imager/Sounder | SSMIS
+    - Microwave Imager/Sounder | SSMIS
```

Comment:
I fiddled with these test responses to exercise a few different "alphabetizing" cases.